docs/Features/Form/Form_Index.md

```diff
@@ -0,0 +1,105 @@
+# Form
+
+Form is a feature that allows you to collect additional data and attach it to created subjects. The supported built-in data is currently:
+
+- Email
+- Text
+
+You can then submit this data, alongside your subject, through the [AddSubjectParameters](../SubjectManagement/SubjectManagement_Index.html#subject-operations).
+
+To start the form feature, you can use the enrolment method
+
+=== "Android"
+    ```kotlin
+    /**
+     * @param context
+     * @param params [FormParameters] required to start the form feature
+     * @param resultLauncher [ActivityResultLauncher<Intent>] fragment or activity that will handle the results
+     */
+    fun startForm(
+        context: Context,
+        params: FormParameters,
+        resultLauncher: ActivityResultLauncher<Intent>
+    )
+    ```
+
+=== "iOS"
+
+    ```swift
+
+    ```
+
+The `FormParameters` holds the flag to indicate wether you want errors to be shown or not, similar to other features
+
+=== "Android"
+
+    ```kotlin
+        /**
+         * @param showErrors flag to show/hide feature errors
+         */
+        data class FormParameters(
+            val showErrors: Boolean,
+        )
+    ```
+
+=== "iOS"
+
+    ```swift
+
+    ```
+
+
+## Handle Result
+
+=== "Android"
+
+    Here's how you can get the result by using the result launcher that's passed as the final parameter:
+
+    ```kotlin
+    private val formCallback: ActivityResultLauncher<Intent> =
+        registerForActivityResult(ShowFormResultLauncher()) {
+            
+        }
+
+    ```
+
+    The result of this operation will be the following model that will either hold an error or an answer
+
+    ```kotlin
+    data class FormActivityResult(
+        val formAnswer: FormAnswer? = null,
+        val formError: FormError? = null
+    )
+    ```
+
+    ```kotlin
+    data class FormAnswer(
+        val formId: String,
+        val answers: List<Answer>
+    )
+    ```
+
+=== "iOS"
+
+    
+
+The FormError has the following structure:
+
+=== "Android"
+    ```kotlin
```

Comment:
Add empty line for formatting reasons (I don't think it works as expected without it)

Reply:
Check other exemples in this file

Reply:
Looks like this to me, I think it's fine.

![Screenshot 2024-03-20 at 10 12 14](https://github.com/vbmobile/mobile-id-documentation/assets/151060466/00b1eb5b-0657-4fc7-beae-dd8c3a6e9cc2)

Reply:
With the added line looks exactly the same. We can add it just to make sure its the same everywhere.